### PR TITLE
new: Added PUT /managed/services/$id

### PR DESCRIPTION
**Pull request by yoshizzle**

### CODE REVIEW

File: src/data/endpoints/managed.yaml [hunk 1]
```diff
@@ -211,6 +211,25 @@ endpoints:
...
+          curl: |
+            curl -H "Content-Type: application/json" \
+                -H "Authorization: Bearer $TOKEN" \
+                -X PUT -d '{
+                    "label": "NewerMonitor",
+                    "service_type": "url",
+                    "address": "http://monitor2.example.org",
+                    "timeout": 60,
+                    "consultation_group": "DevOps",
+                    "region": "us-east-1a",
```
Comment (SzySteve) [inline src/data/endpoints/managed.yaml]: can region be changed?

Comment (yoshizzle) [reply src/data/endpoints/managed.yaml]: yes, if it's a private IP for address

Comment (jamesottinger) [reply src/data/endpoints/managed.yaml]: region should be null in this example

**Comment by yoshizzle** [comment]
added change @jamesottinger asked for regarding null region in the example

**Pull request merged**